utils_pytorch.py

```diff
@@ -79,7 +79,7 @@ def get_model_path(base_path=None, experiment_name=None, make=True):
     return model_path
 
 
-def export_model(model, path=None, input_shape=(1, 3, 64, 64)):
+def export_model(model, path=None, input_shape=(1, 3, 64, 64), use_script_module=False):
```

Comment:
Please default `use_script_module` to `True`.

Reply:
Yes, It should default to True. My mistake as my code uses False always.

For backwards compatibility, as mentioned above the default should be True. 

I agree on this.